components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -6286,6 +6309,8 @@ def reexportedName(name):
                 cgThings.append(CGWrapGlobalMethod(descriptor, properties))
             else:
                 cgThings.append(CGWrapMethod(descriptor))
+                if not descriptor.interface.isIteratorInterface():
```

Comment:
Why can we not generate a DomObjectWrap implementation for iterator interfaces?

Reply:
Sorry for making this mistake. For interfaces that provides Wrap function for both itself and IterableIterator<T> (FormData, Header...), I introduced another trait `DomObjectIteratorWrap` which provides a const `ITER_WRAP` that points to the iterator Wrap function. Then, `IterableIterator<T>::WRAP` could point to `T::ITER_WRAP`, therefore, implements the `DomObjectWrap` trait.